src/rgw/rgw_admin.cc

```diff
@@ -6362,6 +6362,13 @@ int main(int argc, const char **argv)
   }
 
   if (opt_cmd == OPT_ORPHANS_FIND) {
+    if (!yes_i_really_mean_it) {
+      cerr << "accidental removal of active objects can not be reversed; "
```

Comment:
We need to mention that this tool just reports the objects, and trusting its output and deleting objects would be irreversible